api/src/Controllers/AuthenticationController.php

```diff
@@ -119,7 +119,8 @@ private function checkAuthRequiredForSpecificSituations($parts): bool
             ($parts[0] == 'shipment' && $parts[1] == 'containers' && $parts[2] == 'history' && in_array($_SERVER["REMOTE_ADDR"], $bcr)) ||
 
                 # Allow shipping service to update dewar status
-            ($parts[0] == 'shipment' && $parts[1] == 'dewars' && $parts[2] == 'confirmdispatch')
+            ($parts[0] == 'shipment' && $parts[1] == 'dewars' && $parts[2] == 'confirmdispatch') ||
+            ($parts[0] == 'shipment' && $parts[1] == 'dewars' && $parts[2] == 'confirmpickup')
```

Comment:
This might be slightly cleaner

```suggestion
            ($parts[0] == 'shipment' && $parts[1] == 'dewars' && ($parts[2] == 'confirmdispatch' || $parts[2] == 'confirmpickup'))
```